Makefile

```diff
@@ -51,7 +51,6 @@ response_parser.o: response_parser.h response_parser.cpp
 .PHONY: clean, all, test, deploy
 
 deploy:
-	make
```

Comment:
I feel like deploy should make our new web-server executable before deploying it (because what happens if make deploy is called from a cleaned state). let me know your thoughts on that.